exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -226,7 +234,7 @@ def _translate_to_jaeger(spans: Span):
             [
                 _get_long_tag("status.code", status.status_code.value),
                 _get_string_tag("status.message", status.description),
-                _get_string_tag("span.kind", span.kind.name),
+                _get_string_tag("span.kind", _otlp_to_jaeger_span_kind(span.kind)),
```

Comment:
We should be mapping OTLP status values to OpenTracing status values. Most values being the same string with different case is just coincidence or a historical artifact IMO. Also, OTLP defines the values as an enum with numeric values. The name is just a label which may change in future. Considering this, I've replaced `kind.name` with a otlp > jaeger mapping even if they look very similar.